getting-started.md

```diff
@@ -299,3 +299,80 @@ s3-conn   Opaque   1      7m37s
 $ aws s3 ls
 2021-04-12 19:03:32 vela-website
 ```
+
+# For GCP
+
+### Apply Provider configuration
+
+```shell
+$ export GOOGLE_CREDENTIALS=xxx;export GOOGLE_PROJECT=yyy
+
+$ sh hack/prepare-gcp-credentials.sh
+
+$ kubectl get secret -n vela-system
+NAME                                              TYPE                                  DATA   AGE
+gcp-account-creds                                 Opaque                                1      52s
+
+$ k apply -f examples/gcp/provider.yaml
```

Comment:
```suggestion
$ kubectl apply -f examples/gcp/provider.yaml
```

I should have mistaken you for the command `k`, which is short for `kubectl`. Can you correct all of them in this PR or in a new one?

Reply:
Done